RELEASE-NOTES.txt

```diff
@@ -17,6 +17,7 @@
 * [**] You can now create a Site Icon for your site using an emoji. [#16670]
 * [*] Fix notice overlapping the ActionSheet that displays the More Actions in the Editor. [#16658]
 * [*] The quick action buttons will be hidden when iOS is using a accessibility font sizes. [#16701]
+* [*] Block Editor: Improve unsupported block message for reusable block. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3621]
```

Comment:
I've already pointed this out to Carlos in the similar Android PR, but worth ensuring people know that this addition to the release notes is not gonna make it to the AppStore since the Editorial'd notes and their translations are already on their way. I wonder if it's worth a reminder to the team in Slack or something? (also, Peril is supposed to flag this in PRs targeting release that introduce those)

Reply:
Thanks you. Will do 🙇